assets/config.php

```diff
@@ -88,6 +88,10 @@
     'storage_to_config_map' => [ // Used by the TenantConfig feature
         // 'paypal_api_key' => 'services.paypal.api_key',
     ],
+    'identification' => [
+        'header' => 'X-Tenant', // Can be anything, but should really start with "X-",
+        'query_parameter' => '_tenant'
```

Comment:
Do we really need `_tenant` as default query parameter? I saw `tenant` query parameter in couple of real projects before and there were no issues with it.

Reply:
I like the prefix because you will never run into an issue when sending data in your e.g. Vue components. You just set `_tenant` globally for axios.